Zb/meetingroutes (#10)
SocexSolutions/agenda-api: README.md

- [Getting started with docker](https://www.youtube.com/watch?v=gAkwW2tuIqE&t=121s)
- [Express.js Apis](https://www.youtube.com/watch?v=-MTSQjw5DrM)

# Getting Started

### Local API

> For quicker dev we can just `nodemon` to quick restart the api when changes

Reviewer: ```suggestion
For quicker dev you can use `nodemon` to quick restart the api when changes
```
